R/pool.R

```diff
@@ -1,12 +1,47 @@
 
 
-#' TODO
+#' Pool Analysis Results
 #'
-#' @name pool
-#' @param results TODO
-#' @param conf.level TODO
-#' @param alternative TODO
-#' @param type TODO
+#' @param results an analysis object created by [analyse()]
+#'
+#' @param conf.level confidence level of the returned confidence interval.
+#' Must be a single number between 0 and 1. Default is 0.95
+#'
+#' @param alternative a character string specifying the alternative hypothesis,
+#' must be one of "two.sided" (default), "greater" or "less".
+#'
+#' @param type a character string of either `"percentile"` (default) or
+#' `"normal"`. Determines what method should be used to calculate the confidence
+#' intervals. See details.
```

Comment:
"Determines what method should be used to calculate the confidence intervals" -> 
"Determines what method should be used for inference" (since they also give us a p-value and not only CIs?)

Reply:
I think I would prefer to be explicit  i.e. 

> Determines what method should be used to calculate the confidence intervals and p-values 

?

Reply:
ok!